http/task_service.go

```diff
@@ -350,6 +350,19 @@ func (h *TaskHandler) handlePostTask(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	urm := &platform.UserResourceMapping{
+		UserID:       auth.GetUserID(),
+		UserType:     platform.Owner,
+		ResourceType: platform.TasksResourceType,
+		ResourceID:   req.Task.ID,
+	}
+	if err := h.UserResourceMappingService.CreateUserResourceMapping(ctx, urm); err != nil {
+		err = &platform.Error{
```

Comment:
Need to do something with this error. Offhand I don't know what pattern is used in the other parts of the codebase if this call fails.

Reply:
Yah I will look around. I dont know if we delete the newly created task here or just return an error response?